src/main/resources/static/cas1-schemas.yml

```diff
@@ -290,6 +290,10 @@ components:
           description: actual departure date or, if not known, the expected departure date
           type: string
           format: date
+        departureReason:
+          $ref: '_shared.yml#/components/schemas/DepartureReason'
+        departureMoveOnCategory:
+          $ref: '_shared.yml#/components/schemas/MoveOnCategory'
```

Comment:
in an attempt to try and reduce how much unneccessary information we provide on the API i've used the generic 'NamedId' to provide information about premise and apArea  on Cas1SpaceBooking (see line 267). I propose we do the same for these two fields and only add more detailed types if required by the UI.